config/retworkx.sh

```diff
@@ -1,2 +1 @@
-PYTHON_DEPS="setuptools-rust"
-MODULE_RUNTIME_DEPS="rust/1.59.0"
+MODULE_BUILD_DEPS="python-build-bundle rust"
```

Comment:
The python-build-bundle is automatically present, you can in fact remove this dep